cmake/Modules/FindBLIS.cmake

```diff
@@ -0,0 +1,62 @@
+SET(BLIS_INCLUDE_SEARCH_PATHS
+ /usr/include
+ /usr/include/blis
+ /usr/local/include
+ /usr/local/include/blis
+ /usr/local/opt/blis/include
```

Comment:
Is this really needed? Additional packages are installed in usually installed in `/usr/local` or in `/opt` folders (to the best of my knowledge, `/usr/local/opt` is only used internally by Homebrew, but all public packages still need search for headers in either of the above-mentioned dirs
```suggestion
```

Reply:
Made the changes as suggested. Thanks